assets/javascripts/kitten/components/carousel/simple-carousel.js

```diff
@@ -43,21 +49,31 @@ class SimpleCarouselBase extends Component {
   render() {
     const {
       children,
+      containerStyle,
       activePaginationColor,
       paginationColor,
       paginationAlign,
+      paginationStyle,
+      bulletStyle,
     } = this.props
 
     const { totalPagesCount, currentPageNumber } = this.state
     const rangePage = createRangeFromZeroTo(totalPagesCount)
-    const paginationStyle = [
+
+    const containerCustomStyle = [styles.container, { ...containerStyle }]
```

Comment:
On peut se passe de `{ ...`.

```suggestion
    const containerCustomStyle = [styles.container, containerStyle]
```